shared/render/render/XMLRenderer.tsx

```diff
@@ -49,8 +50,11 @@ export const XMLRenderer: Renderer = {
         const visible = (text.match(INITIAL_OP_WITH_PARAGRAPH) || [])[1];
         let paragraph = `<p>${sanitizeStyles(text)}</p>`;
         if (visible) {
-          text = text.replace('{{' + visible + '}}', '');
-          paragraph = `<p if="${escapeXml(visible)}">${sanitizeStyles(text)}</p>`;
+          const visibleTree = Math.parse(visible);
+          if (visibleTree.type === 'OperatorNode') {
```

Comment:
Would be great to add a comment here describing what's going on since it's a bit magical :)